tests/SixLabors.Fonts.Tests/GlyphTests.cs

```diff
@@ -137,11 +137,14 @@ public void RenderWoff2Glyphs_EqualToTtfGlyphs()
             var rendererTtf = new ColorGlyphRenderer();
             TextRenderer.RenderTextTo(rendererTtf, testStr, new RendererOptions(fontTtf)
             {
+                // TODO: Check what is different about the woff font.
+                ApplyKerning = false,
```

Comment:
I'm not sure what is happening here. I did a comparison of the rendered output and the results are the same.

@brianpopow you any ideas?

![ttf](https://user-images.githubusercontent.com/385879/132083522-831cc511-590a-423c-80b5-46edba25a824.png)
![woff2](https://user-images.githubusercontent.com/385879/132083523-4415de84-f38a-4b18-9de7-e877cdf0dd2d.png)

Reply:
@JimBobSquarePants I have just tried this and the test passes. Do i understand this correct, that test fails for you?

Reply:
I’ve turned off kerning in the test. Turning it on leads to different vectors in the results for me.

Reply:
It seems the issue was introduced with d94e0b9e. 

I am not 100% sure yet, but i suspect the change to `AppliedFontStyle.cs`, could be the reason.

Reply:
Thanks yeah, massive rewrites to that section. I’ll be the ttf and woff2 fonts doing different things during glyph substitution but I would have thought the gsub table would be identical in both.

Reply:
yeah I think you are right, there is something different going on during the substitution. Not sure how to fix that yet, but this seems to be the reason for the difference.

Reply:
Woff isn't doing the substitution. The `I` character is being replaced in the TTF version.

Reply:
glad you found this

Reply:
@JimBobSquarePants I would like to change the test a bit and also test for the correct control points count. Is it ok if i commit this change to your branch?

Reply:
Yeah. Absolutely!